src/core/execution/nation/structureSpawnTileValue.ts

```
@@ -54,6 +54,52 @@ export function structureSpawnTileValue(
         }
 
         // TODO: Cities and factories should consider train range limits
+        return w;
+      };
+    case UnitType.SAMLauncher:
+      const structureTiles: TileRef[] = [];
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)